Shared/Managers/LaunchManager.swift

```diff
@@ -48,9 +48,14 @@ class LaunchManager {
             }))
         }
 
+        // Initialize the stores.
+        UserStore.shared.initialize()
+        await ConnectionStore.shared.initialize()
```

Comment:
Initialize both of these manually. The timing is always after parse has become initialized so we don't error out.

Reply:
This was causing the error? lame.